### PR TITLE
WIP: Try to fix ActiveRecord 4.2 on jRuby 9.2.5.0 but no luck yet

### PR DESCRIPTION
**Pull request by Envek**
Tried to solve the problem with JRuby discussed in https://github.com/evilmartians/evil-seed/pull/3#issuecomment-450672080 and https://github.com/evilmartians/evil-seed/issues/6

Still, it doesn't work

### CODE REVIEW

File: lib/evil_seed/refinements/in_batches.rb [hunk 1]
```diff
@@ -7,9 +7,7 @@ module Refinements
     # See https://github.com/rails/rails/commit/25cee1f0373aa3b1d893413a959375480e0ac684
     # The ActiveRecord MIT license is obviously compatible with our license (MIT also)
     module InBatches
-      refine ActiveRecord::Relation do
-        using EvilSeed::Refinements::InBatches
-
+      refine ActiveRecord::Batches do
```
Comment (Envek) [inline lib/evil_seed/refinements/in_batches.rb]: This makes MRI 2.3 to fail as it doesn't want to refine modules

Comment (palkan) [reply lib/evil_seed/refinements/in_batches.rb]: Anyway, it didn't help(

**Comment by palkan** [comment]
Ok, I found it.

This is a known issue (I reported it a while ago 😉): https://github.com/jruby/jruby/issues/5221.

JRuby cannot handle refinements in ancestors 😞

**Comment by Envek** [comment]
Ugh. That means that only choice we have is to drop refinements and use good old monkey patching ¯\_(ツ)_/¯

Thanks for the investigation!

**Comment by palkan** [comment]
>  that only choice we have is to drop refinements and use good old monkey patching 

Why not learning Java and contributing to JRuby?)

**Comment by Envek** [comment]
> Why not learning Java and contributing to JRuby?)

There is also an option to rewrite gem to use SQL cursors and it is much simpler but still a lot of work :sweat_smile:

**Comment by Envek** [comment]
ActiveRecord 4.2 doesn't seem relevant anymore.

**Pull request closed without merging**